source/_integrations/carson.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: Carson
+description: Instructions on how to integrate your Carson.live Account within Home Assistant.
+ha_category:
+  - Camera
+  - Lock
+ha_release: 0.105.0
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@rado0x54'
+---
+
+The `carson` implementation allows you to integrate your [Carson.live](https://carson.live/) account in Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Camera](#camera)
+- [Lock](#lock)
+
+## Configuration
+
+Go to the integrations page in your config and click on new integration -> Carson.
+
+By default, the Carson integration will retrieve accessible cameras from the Carson Interface (e.g. Cameras that you also see within the Carson App).
+However, it is also possible to retrieve the cameras directly via the Eagle Eye API (which Carson uses for Integration), which may allow access to
+more cameras.
+
+## YAML configuration
+
+YAML configuration is around for people that prefer YAML, but it's not preferred!
+
+To enable device linked in your [Carson.live](https://carson.live/) account, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+carson:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+{% configuration %}
+username:
+  description: The username for accessing your Carson account (usually an e-mail).
+  required: true
+  type: string
+password:
+  description: The password for accessing your Carson account.
+  required: true
+  type: string
+{% endconfiguration %}
+
+## Camera
+
+Once you have enabled the [Carson integration](/integrations/carson), you can start using the camera platform.
```

Comment:
Why should we make a link to the same page where the user already is? 🤔

Reply:
No reason, I used https://github.com/home-assistant/home-assistant.io/blob/next/source/_integrations/ring.markdown as a template and they do it like this. Fixed now.